### ISSUE TITLE
Add timestamp to APM log events

### ISSUE THREAD

**Issue by frslater** [F5Networks/f5-telemetry-streaming#91]
As per the "APM Logging" section from the below page of the Telemetry Streaming article (above), the timestamp of the log event is NOT currently being included for APM log events:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
https://clouddocs.f5.com/products/extensions/f5-telemetry-streaming/latest/event-listener.html
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Example Output from Telemetry Streaming:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    "hostname":"telemetry.bigip.com",
    "errdefs_msgno":"01490102:5:",
    "partition_name":"Common",
    "session_id":"ec7fd55d",
    "Access_Profile":"/Common/access_app",
    "Partition":"Common",
    "Session_Id":"ec7fd55d",
    "Access_Policy_Result":"Logon_Deny",
    "tenant":"Common",
    "application":"",
    "telemetryEventCategory":"APM"
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
*  Notice there is no "Timestamp" parameter in the above example from the article. Or in the example the customer provided from their BigIP below:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{ [-]
   Access_Profile: /Common/access-policy-zero-trust-okta-prod
   Client_IP: 192.168.0.22
   Continent:
   Country:
   Listener: /Common/zero-trust-prod-443
   Partition: Common
   Reputation:
   Session_ID: d873f103
   State:
   Virtual_IP: 192.168.1.11
   errdefs_msgno: 01490500:5:
   hostname: vAsm01-wtc.vpn.example.com
   partition_name: Common
   session_id: d873f103
   telemetryEventCategory: APM
   tenant: Common
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

So it seems Telemetry Streaming iControl LX iApp template does not include timestamps within APM logs events.

Steps to Reproduce:
===================

Download, install, configure, and deploy the Telemetry Streaming iControl LX iApp template in BigIP.
Generate APM log events.
Check the remote syslog server for those APM log events and notice they do NOT include a timestamp parameter/value.

**Comment by dstokesf5** [comment]
This has been added to our internal product backlog as AUTOTOOL-2027.

**Comment by cnurao1986** [comment]
Hi Daniel, We are also seeing the same issue, when can we expect the new release with this added feature?

**Comment by dstokesf5** [comment]
@cnurao We are still triaging and prioritizing this issue. We do not have an estimated release at this time.

**Comment by valbuenahector** [comment]
> @cnurao We are still triaging and prioritizing this issue. We do not have an estimated release at this time.

Is there an ETA to resolve this issue ?

**Comment by dstokesf5** [comment]
This issue should be resolved in the 1.17.0 release in mid-January.

**Comment by dstokesf5** [comment]
This has been resolved as part of the 1.17.0 release.
